src/svg.draw.js

```diff
@@ -3,12 +3,12 @@
 
         this.el = el;
         el.remember('_paintHandler', this);
-    
+
         var _this = this,
             plugin = this.getPlugin();
 
         this.parent = el.parent(SVG.Nested) || el.parent(SVG.Doc);
-        this.p = this.parent.node.createSVGPoint(); // Helping point for coord transformation
+        this.p = new SVG.Point(this.parent.node.createSVGPoint()); // Helping point for coord transformation
```

Comment:
I'm not sure if we need to use `node.createSVGPoint()` as constructor argument. In my tests it works fine to use nothing - both results in x:0, y:0. But I did not test with the parent node being transformed. Does it make a difference for `matrixTransform`?

Reply:
Nope you dont need to do that. The reason a native point was used here is that there simply wasnt no SVG.Point when I wrote this plugin :).